addon/services/tour.js

```diff
@@ -154,60 +168,21 @@ export default Service.extend(Evented, {
       //Style all internal elements as well
       const children = $(currentElement).children();
       const clonedChildren = highlightElement.children();
+
       for (let i = 0; i < children.length; i++) {
-        clonedChildren[i].style.cssText = window.getComputedStyle(children[i]).cssText;
+        clonedChildren[i].style.cssText =
+          window.getComputedStyle(children[i]).cssText;
       }
-      highlightElement.css('position', 'absolute');
-      highlightElement.css('left', elementPosition.left);
-      highlightElement.css('top', elementPosition.top);
-      highlightElement.css('width', elementPosition.width);
-      highlightElement.css('height', elementPosition.height);
-      highlightElement.css('z-index', '10002');
-    }
-  },
 
-  /**
-   * Exchanges the given attachment configuration for the object equivalent with
-   * the DOM element passed in directly.
-   *
-   * Allows for use of pseudoselectors, because it uses jQuery's selector engine
-   * instead of the built-in one
-   *
-   * @param {Object} attachTo The given "attachTo" configuration
-   * @return {Object} The optimized configuration
-   * @private
-   */
-  exchangeForAttachmentConfig(attachTo) {
```

Comment:
We need to support passing a string, an object, or an actual DOM object through to shepherd, as per the docs:

```
attachTo: What element the step should be attached to on the page.
It can either be a string of the form "element on", or an object with 
those properties. For example: ".some #element left", or 
{element: '.some #element', on: 'left'}. If you use the object syntax,
element can also be a DOM element. If you don't specify an 
attachTo the element will appear in the middle of the screen.
```